app/models/concerns/hyrax/ability.rb

```diff
@@ -188,5 +193,15 @@ def user_is_depositor?(document_id)
       def curation_concerns_models
         [::FileSet, ::Collection] + Hyrax.config.curation_concerns
       end
+
+      def can_review_submissions?
+        # Are there any workflows where this user has the "approving" responsibility
+        approving_role = Sipity::Role.find_by(name: 'approving'.freeze)
```

Comment:
I'm not a fan of hard-coding a magic role in an opaque ability class. I'm thinking that we need a declaration of "magic roles" in the application: :admin, :superadmin, :approving, :depositing.

I also believe that there should be a method in the PermssionQuery that handles this query without requiring looping through.